oeps/oep-0018-bp-python-dependencies.rst

```diff
@@ -208,14 +234,23 @@ this file:
   constraint has been imposed.  If there is an issue (either in Jira or an
   upstream issue tracker) for resolving the problem, a link to it should be
   included in the comment.
-* Minimum versions should generally not be included here; ``pip-compile``
-  always tries to use the latest compatible version in the generated
-  requirements files.  If minimum versions need to be specified for use in
-  ``setup.py``, those constraints should go in ``requirements/base.in`` as
-  explained above.
-
-This file should be periodically reviewed to determine if some of the
-constraints are no longer required.
+* Minimum constraints are required in some circumstances, and should be avoided
+  in others.
+
+  * For IDAs, minimum versions are generally not necessary since ``pip-compile``
+    always tries to use the latest compatible version in the generated
+    requirements files; if ``make upgrade`` produces a downgrade in the compiled
+    dependencies, it should be carefully investigated before committing.
+  * For library packages, minimum versions should be specified when a committer
```

Comment:
I wonder if min versions even deserve a comment. Mostly they should just get ratcheted up, and if someone wants to know why it's specified, it's likely because of a recent change which broke a relying package (and therefore should be easy to find in git history.)

Reply:
I agree that the comment is useful for max constraints, and doesn’t seem necessary for minimum constraints that will never Ned to be removed.